lane.py

```diff
@@ -693,7 +693,11 @@ def modify_database_query(self, _db, qu):
             # depending on what exactly the wording is.
             availability_clause = LicensePool.open_access == True
         elif self.availability == self.AVAILABLE_NOT_NOW:
-            availability_clause = not_(available_now)
+            # The book must be licensed but currently unavailable.
+            availability_clause = and_(
+                not_(available_now),
+                LicensePool.licenses_owned > 0
+            )
```

Comment:
This does make a difference; without this code a book that's _never_ going to be available could show up in a feed of  books that are not available _right now_.

Reply:
In what case will a book be stored that will never be available? (If a book is not getting a license, why include it in the db?)

Reply:
This happens when a library licenses a book and then the license expires. We still want to keep track of the book because people may have active loans or annotations on the book. 

"Never" is maybe too strong because the library could always license the book again.